akka-remote/src/test/scala/akka/remote/artery/RemotingFlightRecorderSpec.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2009-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.remote.artery
+
+import akka.actor.ActorSystem
+import akka.testkit.TestKit
+import org.scalatest.Matchers
+import org.scalatest.WordSpec
+
+class RemotingFlightRecorderSpec extends WordSpec with Matchers {
+
+  "The RemotingFlightRecorder" must {
+
+    "use the no-op recorder by default" in {
+      val system = ActorSystem("RemotingFlightRecorderSpec")
+      try {
+        val extension = RemotingFlightRecorder(system)
+        extension should ===(NoOpRemotingFlightRecorder)
```

Comment:
Because of the current setup that class won't be available. I tried adding a separate `jdk-9-only` test suite like `akka-stream` has but that doesn't seem to work. I think we'll have to take a closer look at how we do the JDK9-only stuff and see if we can A. get it to work at all as expected B. make it less surprising.

Reply:
Got it to work, now has JDK 11 coverage in `src/test/scala-jdk9-only/akka/remote/artery/jfr/JFRRemotingFlightRecorderSpec.scala`